src/vs/workbench/api/common/extHostEditorTabs.ts

```diff
@@ -94,6 +94,8 @@ class ExtHostEditorTab {
 				return new NotebookDiffEditorTabInput(URI.revive(this._dto.input.original), URI.revive(this._dto.input.modified), this._dto.input.notebookType);
 			case TabInputKind.TerminalEditorInput:
 				return new TerminalEditorTabInput();
+			case TabInputKind.InteractiveEditorInput:
+				return new InteractiveWindowInput(URI.revive(this._dto.input.uri), URI.revive(this._dto.input.inputUri));
```

Comment:
The `uri` represents the embed notebook resource (which is used as execution history) while `inputUri` represents the input box (rendered in monaco editor) at the bottom of the interactive window. Here I call it `uri` directly since the history notebook is the main resource in this embed editor (internally IW is being treated as an embed notebook editor most of the time).